support/config/deployment.go

```diff
@@ -346,3 +365,7 @@ func debugDeployments(hc *hyperv1.HostedControlPlane) sets.String {
 	names := strings.Split(val, ",")
 	return sets.NewString(names...)
 }
+
+func emptyDirVolumeAggregator(volumes ...string) string {
+	return fmt.Sprintf("\"%s\"", strings.Join(volumes[:], ","))
```

Comment:
why do we need this? why not just use `strings.Join()` directly?

Reply:
Fixed